CONTRIBUTING.md

```diff
@@ -85,6 +85,35 @@ git push origin my-branch
 
 5. Open [this project on GitHub](https://github.com/theforeman/foreman-js), then click “Compare & pull request”.
 
+## Testing Pull Requests
+
+1. Open pull request in Github and go to the tasks tab.
+2. Expand `Build tarballs for testing` jobs and open the relevant action. For example, the job for the test package would be `build-tarball (test)`.
+3. Expand the artifacts dropdown on the top right and download the appropiate artifact. For example, the test package will be titled `foreman-js-test-pr-###.tgz`.
+4. Unzip and copy the unpacked tarball to your development box or the appropiate location:
+```bash
+# download `foreman-js-test-pr-123.tgz.zip` from `artifacts` dropdown on top right of github action
+cd /path/to/download_dir && unzip foreman-js-test-pr-123.tgz.zip
+scp theforeman-test-4.15.1.tgz vagrant@centos7-katello-devel-stable.example.com:~
+```
+  - Side note: ideally we could use the tarball URL directly but looks like there is a permission issue that will make it hard to use with npm: https://github.com/actions/upload-artifact/issues/51
```

Comment:
Would it work to have a shell script download the tarball with `curl`?
If so, it wouldn't be too hard to have a script that automates steps 4 through 8-- you could just pass in the url.

Reply:
From the issue, it sounds like `curl -v -H "Authorization: token $GITHUB_TOKEN" $TARBALL_URL` would work

We could try out the curl command and give it as an option if it works as expected. I would just want to make sure the permissions don't block it for non-maintainers (or we make that distinction clear in the docs)